Makefile

```diff
@@ -386,7 +386,6 @@ force_symbols:
 	$(PYTHON) ./tools/symbols.py map build/us/ric.map --no-default > config/symbols.us.ric.txt
 	$(PYTHON) ./tools/symbols.py map build/us/stcen.map --no-default > config/symbols.us.stcen.txt
 	$(PYTHON) ./tools/symbols.py map build/us/stdre.map --no-default > config/symbols.us.stdre.txt
-	$(PYTHON) ./tools/symbols.py map build/us/stmad.map --no-default > config/symbols.us.stmad.txt
```

Comment:
Is this intentionally removed?

Reply:
Yep! We do not have `us` for MAD. The only "version" that exists for MAD is `beta`. I put it there by mistake long time ago